compiler/src/typed/types.re

```diff
@@ -71,10 +71,13 @@ and abbrev_memo =
   // Abbreviations can be found after this indirection
   | TMemLink(ref(abbrev_memo));
 
+/*true if a constant false if a block*/
```

Comment:
ADT might be better, idk

Reply:
Yeah, I had that thought, but didn't know if it was worth it. Do you want me to change it?